presto-cassandra/src/main/java/com/facebook/presto/cassandra/CachingCassandraSchemaProvider.java

```diff
@@ -176,7 +179,7 @@ public void flushCache()
         return ImmutableList.copyOf(getCacheValue(schemaNamesCache, "", RuntimeException.class).keySet());
     }
 
-    private Map<String, String> loadAllSchemas()
+    private ImmutableMap<String, String> loadAllSchemas()
```

Comment:
It is required for the Luna compiler; it seems to be a bug, the Mars compiler understands this correctly.